Libraries/RhythmGamesBot.Core/RhythmGamesBot.cs

```diff
@@ -8,7 +8,7 @@ namespace RhythmGamesBot.Core
 {
     public class RhythmGamesBot
     {
-        private static Option<DiscordClient> BotInstance { get; set; }
+        private static DiscordClient BotInstance { get; set; }
```

Comment:
oh wait it's static, why is it static? then either make the class and the constructor static or the property non-static

Reply:
Resolved in recent commit